Servers/swagger.yaml

```
@@ -3751,8 +3751,158 @@ paths:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: ok
+                  data:
+                    type: object
+        "401":
+          description: Unauthorized
+        "500":
+          description: Internal server error
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)